### ISSUE TITLE
Phone signup without email recovery step

### ISSUE THREAD

**Issue by sonyarouje** [Azure-Samples/active-directory-b2c-custom-policy-starterpack#64]
I uploaded phone-number-passwordless custom policy and features are working as expected. 
But I wanted to remove email recovery step when a user signup using phone number. 

As per the current scenario a user need to verify email id as well, even when doing a phone number based sign up.

Any help will be really appreciated.

**Comment by dapug** [comment]
Super dumb. There are valid cases to NOT use email. And, allow me to highlight a keyword here: SignUpOrSignInWithPhone**Or**Email.  Yeah, says Or, not AND.  Ugh.

Anyway, while we wait for Microsoft to make developing custom policies possible for mere mortals, maybe even without requiring living in the matrix with multi-thousand line XML files, I found this post that may be helpful:
https://docs.microsoft.com/en-us/answers/questions/115660/azure-ad-b2c-sign-up-with-phone.html

**Comment by HappyNomad** [comment]
@dapug That looks like a good resource to get started with, thanks!

@sonyarouje Is that the approach you took or is there another option?

**Comment by VitikaSoni** [comment]
Hey, does anyone know a solution to this? The answer in [this post](https://learn.microsoft.com/en-us/answers/questions/115660/azure-ad-b2c-sign-up-with-phone) is a little confusing and hard to follow, as I want only phone+otp signup up (no social account or MFA).
